client/src/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { FiMenu } from "react-icons/fi";
+import { MdClose } from "react-icons/md";
+import styles from "./PageHeader.module.css";
+import { PropTypes } from "prop-types";
+
+const MenuDropdown = ({ onClose }) => {
+  const navigate = useNavigate();
+  const handleLogOut = () => {
+    // TODO : API call for logout
+    navigate("/");
+  };
+
+  return (
+    <div className={styles.navDropdown}>
+      <NavButton hamburger={false} onClick={onClose} />
+      <div className={styles.menuItemContainer}>
+        <MenuDropdownItem
```

Comment:
Although this is fine, an alternative that we can have is to have a list of objects such as 
```
[{label: "Calendar", to: "/calendar"}, {label: "Task History", to: "/task_history"}, ...]
```
so that we can render these buttons dynamically if we were to add in more pages/buttons